.env

```diff
@@ -60,3 +60,7 @@ HASH_TELEMETRY_DESTINATION=REPLACE_ME.aws.com
 HASH_TELEMETRY_APP_ID=hash-workspace-app
 
 ORY_KRATOS_PUBLIC_URL=http://127.0.0.1:4433
+
+ORY_KRATOS_ADMIN_URL=HTTP://127.0.0.1:4434
+
+KRATOS_API_KEY=secret
```

Comment:
So this `.env` is not bound to the actual secret in the file [here](https://github.com/hashintel/hash/blob/c04523804d92f74eaf3a439fb66f82fc81525894/packages/hash/external-services/kratos/kratos.dev.yml#L51)?

Reply:
Correct, it is used in the workspace backend however.